javaparser-core-testing/src/test/java/com/github/javaparser/PositionTest.java

```diff
@@ -0,0 +1,23 @@
+package com.github.javaparser;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PositionTest {
+
+    @Test
+    public void testOrIfInvalid() {
+        Position p1 = new Position(1, 1);
```

Comment:
There are only tests for the orIfInvalid method does this mean that the other modified methods are sufficiently tested?

Reply:
Yes, the IntelliJ code coverage tools show that tests hit the other methods in position. If there is any particular you would like tested we are happy to write more tests.